proj/src/components/projects/EditProject.js

```diff
@@ -1,5 +1,10 @@
-import React from 'react'
-import styled from 'styled-components'
+import React from 'react';
+import styled from 'styled-components';
+import { connect } from 'react-redux';
+import uuidv4 from 'uuid/v4';
+import PropTypes from 'prop-types';
+
+import { closeEditModal, isLoading, addProject, editProject} from 'actions/projectsActions';
```

Comment:
you can use import * as projectActions here if you have to import many actions. it also prevent the confusion of using the imported action creator or dispatching the action from props